m3/include/m3/thrift/M3.h

```diff
@@ -0,0 +1,244 @@
+/**
```

Comment:
could you adding a folder/ns to indicate this is generated code ( `gen`/`generated`/etc)

Reply:
`s/thrift/gen`?

Reply:
check if there's an existing cpp convention; `tally-cpp/whatver/parent/gen/thrift/...`  maybe?

Reply:
I'm sure there's at least 7 different conventions with C++ :) I'll wait for the ATG guys to review and we can follow their convention.

Reply:
lol 👍

Reply:
Ya don't assume any C++ is normative. Styles vary for everything.